urlhaus/connector.py

```diff
@@ -0,0 +1,30 @@
+"""
+Copyright start
+Copyright (C) 2008 - 2021 Fortinet Inc.
```

Comment:
please change the copyright year to 2008 - 2023 in each file

Reply:
Change has been done